src/js/tabs/receiver.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+var CryptoJS = require("crypto-js");
```

Comment:
@chmelevskij are we going to use require or move this somewhere else?

Reply:
Ah, looks like most `require`s are in gulpfile.js. Should I move it there?

Reply:
Avoid imports if possible please 🙏